edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
         },
         updateRemainingTime: function (self) {
+            self.timerTick ++;
+            if (self.timerTick % 5 == 0){
+                var url = self.model.url + '/' + self.model.get('attempt_id');
+                $.ajax(url).success(function(data) {
+                    if (data.status === 'error') {
+                        // Let the student know that his exam has failed due to an error.
+                        // This alert may or may not bring the browser window back to the
+                        // foreground (depending on browser as well as user settings)
+                        alert(gettext('Your exam has failed'));
```

Comment:
In this case the exam is stopped and - from an internal point of view - mark in error. This is an alert with just an OK button. Clicking OK will just cause the page to refresh so that the exam content will be blocked from the user being able to interact with it.

@pbaruah @griffresch from a product point of view